iron-list.html

```diff
@@ -989,6 +1064,139 @@
           this.scrollToIndex(this.firstVisibleIndex);
         }
       });
+    },
+
+    _getModelFromItem: function(item) {
+      var key = this._collection.getKey(item);
+      var pidx = this._physicalIndexForKey[key];
+
+      if (pidx !== undefined) {
+        return this._physicalItems[pidx]._templateInstance;
+      }
+      return null;
+    },
+
+    /**
+     * Select the list item at the given index.
+     *
+     * @method selectItem
+     * @param {Object} item the item object or its index

```

Comment:
Use `|` syntax for multiple-type args:
```
@param {(Object|number)} item the item object or its index
```

https://developers.google.com/closure/compiler/docs/js-for-compiler?hl=en#types